config.py

```diff
@@ -233,6 +233,16 @@ class Config(object):
             ReturnCodes.TEMPORARY,
             (True, True)
         ),
+        'learning.mozilla.org': (
+            'https://foundation.mozilla.org/en/opportunity/web-literacy/',
+            ReturnCodes.TEMPORARY,
+            (True, True)
+        ),
+        'teach.mozilla.org': (
+            'https://foundation.mozilla.org/en/opportunity/web-literacy/',
+            ReturnCodes.TEMPORARY,
+            (True, True)

```

Comment:
Let's not preserve the path portion of the request.
```suggestion
            (False, True)
        ),
        'teach.mozilla.org': (
            'https://foundation.mozilla.org/en/opportunity/web-literacy/',
            ReturnCodes.TEMPORARY,
            (False, True)

```